src/app/shared/Feed.svelte

```diff
@@ -240,10 +240,10 @@
   </FeedControls>
 {/if}
 
-<FlexColumn xl bind:element>
+<FlexColumn bind:element>
   {#each events as note, i (note.id)}
     <div in:fly={{y: 20}}>
-      <Note {filters} {reposts} {depth} {anchor} {note} />
+      <Thread {filters} {reposts} {depth} {anchor} {note} />
```

Comment:
Thread sounds like ThreadDetail, which is a different kind of thing. Let's name this `FeedItem`